src/js/components/DataSearch/DataSearch.js

```diff
@@ -19,12 +24,15 @@ export const DataSearch = ({ ...rest }) => {
       id={`${dataId}--search`}
       name="_search"
       icon={<Search />}
+      reverse={addSubmit}
       type="search"
       {...rest}
     />
   );
 
-  if (noForm) content = <DataForm footer={false}>{content}</DataForm>;
+  if (noForm) {
```

Comment:
Any reason this line changed? Seems like we can keep in the simplified format from before.

Reply:
reverted to simplified syntax